test/unit/carousel.tests.js

```diff
@@ -451,4 +474,33 @@ describe('Carousel', () => {
       expect(document.getElementById('slide10')).to.exist;
     });
   });
+
+  it('should render custom arrow', done => {
```

Comment:
Can you add a unit test about what happens when no `customArrow.className` is provided